scripts/kind-cluster-with-registry.sh

```diff
@@ -18,6 +18,25 @@ fi
 cat <<EOF | kind create cluster --image kindest/node:v1.21.1 --name "${KIND_CLUSTER_NAME}" --config=-
 kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  kubeadmConfigPatches:
+  - |
+    kind: InitConfiguration
+    nodeRegistration:
+      kubeletExtraArgs:
+        container-log-max-size: "100Mi"
```

Comment:
I am a bit torn about this, will it fill the artifacts sooner?
Maybe we should have this "high verbosity" mode optional under some env variable, so if we are not able to debug CI we'll add a commit raising that flag. WDYT?

Reply:
Commenting here what we talked about - we decided to keep it this way since it produces a log file of about ~50MB and only when the CI fails. If any issues with artifacts size arise we will rethink this matter.